scripts/run.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 export PULSE_SERVER=unix:/run/user/`id -u`/pulse/native
-export PIPEWIRE_RUNTIME_DIR=unix:/run/user/`id -u`
+#export PIPEWIRE_RUNTIME_DIR=unix:/run/user/`id -u`
```

Comment:
Let's just remove it, rather than comment it out.

Reply:
Done.